envs/example/defaults.yml

```diff
@@ -322,15 +323,15 @@ keystone:
     - name: heat
       type: orchestration
       description: 'Heat Orchestration API'
-      public_url: https://{{ endpoints.main }}:8005/v1/%(tenant_id)s
-      internal_url: https://{{ endpoints.main }}:8005/v1/%(tenant_id)s
-      admin_url: https://{{ endpoints.main }}:8005/v1/%(tenant_id)s
+      public_url: https://{{ endpoints.main }}:8004/v1/%(tenant_id)s
```

Comment:
CJ, let make this PR without heat ports changes (i.e uses old heat ports).  Work #1144  should cover heat port standardization.